source/backend/database/models/savedRecipesModel.js

```diff
@@ -30,12 +31,22 @@ const db = require('../dbConfig');
  * @param userId
  * @returns {Promise<awaited Knex.QueryBuilder<TRecord, TResult>>}
  */
- async function getSavedRecipes(userId){
+ async function getSavedRecipes(userId) {
```

Comment:
ditto. since this has a join, the name should be:`getByUserIdJoinRecipes`

Reply:
Fixed